src/HotChocolate/AspNetCore/test/AspNetCore.Tests/HttpPostMiddlewareTests.cs

```diff
@@ -778,5 +778,73 @@ id @export
             // assert
             result.MatchSnapshot();
         }
+
+        [Fact]
+        public async Task Fragments()
+        {
+            // arrange
+            TestServer server = CreateStarWarsServer();
+
+            // act
+             ClientQueryResult result =
```

Comment:
This now uses a full test server and a POST request.

Reply:
As discussed in Slack, I think this is probably unnecessary now, as my commit 714c8b1 corrects the test. It was `if: true` that causes the problem. With that in place you should be able to repro with just my test. Sorry, fail on my part !